src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/File/Copy.cs

```diff
@@ -352,6 +363,8 @@ public void WindowsAlternateDataStreamOverwrite(string defaultStream, string alt
 
             // Copy the default stream into an alternate stream
             File.WriteAllText(testFileDefaultStream, "Foo");
+            Assert.Equal("Foo", File.ReadAllText(testFileDefaultStream));
+            Console.WriteLine($"Copying default stream into alternate stream.");
```

Comment:
I am not sure if the output written to the console will be accessible in the logs. IIRC the xUnit recommended approach is to add a public ctor that accepts `ITestOutputHelper`, store it in the field and then use it for writing in tests.

Sth like here:

https://github.com/dotnet/runtime/blob/d2a2a79b2eaf66d68a02f35917907bd9fbe97ea7/src/libraries/System.Private.Xml/tests/Xslt/XslTransformApi/CThreads.cs#L18-L21

Reply:
Changed to use `ITestOutputHelper` and it works.